index.html

```diff
@@ -15,19 +15,23 @@ <h2>Overview</h2>
          <table>
     <tr>
         <td><img src="https://mk0itemsssrcorgqb419.kinstacdn.com/wp-content/uploads/2019/03/iStock-165790610-1000x569.jpg" width="350" height="300"></td>
-        <td><h2>What is Thinkment?</h2> <p>Thinkment is a philosphical and ideology based website that focuses on describing the pros/cons, political influence, systematic practice, and connections between Socialism and Capitalism being controversial topics. Philosophy will further look at the influence of the age of enlightment, the Roman influence of early Afro-European politics and nationalites.</p></td>
+        <td><h2>What is Thinkment?</h2> <p>Thinkment is a philosphical and ideological based website that focuses on describing the pros/cons, political influence, systematic practice, and connections between Socialism and Capitalism being controversial topics. Philosophy will further look at the influence of the age of enlightment, the Roman influence of early Afro-European politics and nationalites.</p></td>
     </tr>
 </table>
          <table>
     <tr>
         <td><img src="https://d3n8a8pro7vhmx.cloudfront.net/democracyatwork/pages/947/meta_images/original/capitalism_socialism_thumb.png?1481377102" width="350" height="300"></td>
         <td><h3>Philosophical Issues</h4>
-        <p> Communism and Capitalism are two different theorys that orrespond how a societal structure should work. Communism in summary focuses on the eqaality of wealth and the control of production under workers. (More on industrilization). Capaitalism focuses more on a wealth class system, and the control of capital between competition and power.</p></td>
+        <p> Communism and Capitalism are two different political theorys that correspond how a societal structure should work. Communism in summary focuses on the equality of wealth and the control of production under workers. (More on industrilization). Capitalism focuses more on a wealth class system, and the control of capital between competition and power.</p></td>
```

Comment:
`theorys` -> `theories`.

`that correspond how` -> `about`

Why is ` (More on industrilization). ` in parentheses in a standalone sentence? Is it part of the prior sentence?